src/serverenvironment.cpp

```diff
@@ -1210,9 +1211,6 @@ void ServerEnvironment::clearObjects(ClearObjectsMode mode)
 		m_script->removeObjectReference(obj);
 
 		// Delete active object
-		if (obj->environmentDeletes())
-			delete obj;
```

Comment:
The comment now looks obsolete

Reply:
It's for the `return`. See comment in server/activeobjectmgr.h.